_migration-assistant/deploying-migration-assistant/configuration-options.md

```diff
@@ -100,7 +100,8 @@ The following sample CDK performs a live capture migration with C&R:
     "captureProxyServiceEnabled": true,
     "captureProxyExtraArgs": "",
     "trafficReplayerServiceEnabled": true,
-    "trafficReplayerExtraArgs": "",
+    "trafficReplayerExtraArgs": "--speedup-factor 10.0",
+    "targetClusterProxyServiceEnabled": true,
     "artifactBucketRemovalPolicy": "DESTROY"
```

Comment:
Can we remove any of these bucket policies in the docs?  We shouldn't have these bleed through to our users.